packages/driver/src/cypress/mocha.ts

```diff
@@ -490,7 +458,28 @@ function createCalculateTestStatus (Cypress: Cypress.Cypress) {
   Test.prototype.calculateTestStatus = function () {
     const retriesConfig = getNormalizedRetriesConfig(Cypress)
 
-    return calculateTestStatus(this, retriesConfig)
+    //@ts-expect-error
+    const actions = Cypress.__actions
+
+    const action = actions?.find((a) => a.clientId === this.id && a.action === 'BURN_IN') ?? {}
+
+    const latestScore = action?.payload?.startingScore === undefined ? -2 : action?.payload?.startingScore
+
+    // console.log('latestScore', latestScore)
+
+    const userFacingBurnInConfig = Cypress.config('experimentalBurnIn')
+
+    // console.log('userFacingBurnInConfig', userFacingBurnInConfig)
+
+    const localBurnInConfig = getBurnInConfig(userFacingBurnInConfig)
+
+    // console.log('localBurnInConfig', localBurnInConfig)
+
+    const completeBurnInConfig = mergeBurnInConfig(action?.payload?.config ?? {}, { values: localBurnInConfig })
+
+    // console.log('completeBurnInConfig', completeBurnInConfig)
```

Comment:
Can these commented-out console logs be removed?

Reply:
These comments are now removed